src/hotspot/share/utilities/numberSeq.hpp

```diff
@@ -47,7 +47,7 @@ class AbsSeq: public CHeapObj<mtInternal> {
   void init(double alpha);
 
 protected:
-  int    _num; // the number of elements in the sequence
+  uint    _num; // the number of elements in the sequence
```

Comment:
All `int` -> `uint` changes in this class/file are good; can they be extracted out to its own PR/ticket?